### DISCUSSION TITLE
Company profile issue

### DISCUSSION THREAD

**Discussion by Kwaku-Ananse** [cytech/BillingTrack#53] [General]
Adding any text in the the fax number, mobile number ID and VAT number fields does not have any effect on the invoice...none of those will appear on invoice..

email address you put in company profile also doesn't reflect on invoice, instead, the email address of admin user appear on invoice.

I could add missing info on invoice for company in the system settings address format,...VAT number, ID number etc can be added to company profile on final out of invoice but then, the same info added also appear on the side of the client / customer. Meaning, if i add my VAT number, it also appear as if it is the vat id for the client being billed.

I changed number of decimals for tax rounding to 2 but there are still 000 (13.000%).....this could be 0 decimals like the quantity and amounts.

there is blank avatar for user profile but no option to replace it in settings....no option in settings to replace main logo

Editing any ALREADY PAID or ALREADY APPROVED QUOTE make it return to a fresh state where you have to resend invoice, then record payment again, doing any of this results in the invoice showing up in the list of payments once again. ...basically, you get the same quote or payment twice. ...you have have to delete duplicates

saving an invoice, quote, estimate, or recording a payment should exit from that current editing mode and take you to a list of invoices, or quotes...whatever you edited.....remaining in the edit mode after saving a payment or an already sent invoice / quote will or could somewhat trigger a duplicate...same with work orders....

Setting up a reminder or an appointment has now email alert or any form of alert while you are still signed in 

When adding a new invoice, the customer field doesn't auto populate existing client when you click inside the field, list or client's name comes up as you type their name

I know these are minor issues...would be great if looked at and resolved in next update

**Comment by cytech** [comment]
YOU need to modify the invoice template for your needs
see https://github.com/cytech/BillingTrack/blob/master/app/Modules/Documentation/Views/documentation/Invoice-Templates.md

**Reply by Kwaku-Ananse** [reply]
no success....i replied to your email few week ago for help please
